res/database/Default/database_news.xml

```diff
@@ -9205,5 +9059,222 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
+			</description>
+			<variables>
+				<actress>${fayedunaway_or_pamelaanderson}|${farrahfawcett_or_winonaryder}|${kirstiealley_or_drewbarrymore}|${kimbasinger_or_elizabethberkley}|${jennifergrey_or_rosemcgowan}|${jennifercoolidge_or_tarareid}|${tatumoneal_or_parishilton}|${brigittenielsen_or_siennamiller}|${robingivens_or_lindsaylohan}</actress>
+				<fayedunaway_or_pamelaanderson>${.lte:${.worldtime:"year"}:1990:"${.person:"c73d7fd8-658f-4638-b3bd-515dc4e76d43":"fullname"}":"${.person:"234ba25b-ea07-427c-92d0-5c475c673ccf":"fullname"}"}</fayedunaway_or_pamelaanderson>
+				<farrahfawcett_or_winonaryder>${.lte:${.worldtime:"year"}:1992:"${.person:"300400b0-a0d9-44e0-a9b2-3fe092ad57ac":"fullname"}":"${.person:"ba3717c8-1685-464b-84c7-5ea7db5c27cc":"fullname"}"}</farrahfawcett_or_winonaryder>
+				<kirstiealley_or_drewbarrymore>${.lte:${.worldtime:"year"}:1995:"${.person:"DF-Celebreties-Kirstie_Alley":"fullname"}":"${.person:"c28d2a5d-2d88-44fa-ba08-7c62b4493c25":"fullname"}"}</kirstiealley_or_drewbarrymore>
+				<kimbasinger_or_elizabethberkley>${.lte:${.worldtime:"year"}:1996:"${.person:"3342a0e3-66f3-4f30-922c-ebe1b0611a00":"fullname"}":"${.person:"382dc521-a101-41a1-aea1-e7d9af52f4c9":"fullname"}"}</kimbasinger_or_elizabethberkley>
+				<jennifergrey_or_rosemcgowan>${.lte:${.worldtime:"year"}:1997:"${.person:"78ca5f5f-3759-4e5b-84f5-e94afafe8f53":"fullname"}":"${.person:"071d38db-704f-4dff-881d-9296ee6765b9":"fullname"}"}</jennifergrey_or_rosemcgowan>
+				<jennifercoolidge_or_tarareid>${.lte:${.worldtime:"year"}:1998:"${.person:"b894f32d-1b5e-4fa5-9eed-b6001054174d":"fullname"}":"${.person:"be2c3d94-5d6e-4302-a39e-a3172c92b95d":"fullname"}"}</jennifercoolidge_or_tarareid>
+				<tatumoneal_or_parishilton>${.lte:${.worldtime:"year"}:2001:"${.person:"bc62084c-e611-4fbf-aafb-ae40e4d0f380":"fullname"}":"${.person:"94ca6c17-441a-4540-a323-fccbd20af333":"fullname"}"}</tatumoneal_or_parishilton>
+				<brigittenielsen_or_siennamiller>${.lte:${.worldtime:"year"}:2001:"${.person:"4a295a3c-6b4f-4b7f-b64f-12351fcdaa30":"fullname"}":"${.person:"1d23e254-2358-4987-87f9-a95a13b21b54":"fullname"}"}</brigittenielsen_or_siennamiller>
+				<robingivens_or_lindsaylohan>${.lte:${.worldtime:"year"}:2006:"${.person:"1cce02b2-ff16-4e2a-9c62-cbade40d0b4a":"fullname"}":"${.person:"38c85dc0-2189-452f-85a4-676d99b9b9ec":"fullname"}"}</robingivens_or_lindsaylohan>
+				<alcoholvalue>
+					<de>1,3|1,4|1,5|1,6|1,7|1,8|1,9|2,0|2,1</de>
+					<en>.13|.14|.15|.16|.17|.18|.19|.20|.21</en>
+				</alcoholvalue>
+			</variables>
+			<data genre="1" price="1" quality_min="30" quality_max="60" fictional="1" />
+			<effects>
+				<!-- TODO: Similarly as previous entries: Was probably intended to make use of popularity trigger feature, but it needs to be applicable to actor chosen by variable, and questionable if it makes a big difference after all -->
```

Comment:
For a trigger to work, not the full name of the actor would have to be stored in the variable, but rather the GUID. Maybe you want to refactor the news accordingly?

Reply:
yes but does the `guid` field of a trigger resolve variables and expressions?